pkg/sqlite/sql.go

```diff
@@ -106,9 +107,11 @@ func getSearchBinding(columns []string, q string, not bool) (string, []interface
 		notStr = " NOT"
 		binaryType = " AND "
 	}
-
+	q = strings.TrimSpace(q)
+	q = regexp.MustCompile(`\s+`).ReplaceAllString(q, " ")
 	queryWords := strings.Split(q, " ")
 	trimmedQuery := strings.Trim(q, "\"")
+
 	if trimmedQuery == q {
```

Comment:
If we relax a bit on the "exact" query this is ok.
If "exact" though includes spaces then this is not ok and test `q = "   \"Girl     does   threesome\" "` (multiple spaces in the quoted query vs single in the title) should fail
If the later is what we want then the 
```golang
q = regexp.MustCompile(`\s+`).ReplaceAllString(q, " ")
queryWords := strings.Split(q, " ")
```
should be moved inside the 
```golang
if trimmedQuery == q {
```

Reply:
You are right, exact should be exactly what the user want to type.